src/main/java/com/harrison/Application.java

```diff
@@ -1,6 +1,7 @@
 package com.harrison;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.SpringApplication;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)